File: testdata/MoveMethod/Java/current/OrderProcessor.java
import java.util.List;

public class OrderProcessor {

    private List<Item> items;

    public void processOrder() {
        double total = calculateTotal(items);
        // Further order processing
    }
}

class Item {
    private double price;

    public double getPrice() {
        return price;
    }

    public static double calculateTotal(List<Item> items) {
        double total = 0;
        for (Item item : items) {
            total += item.getPrice();
        }
        return total;
    }
}

File: testdata/MoveMethod/Cpp/current/OrderProcessor.cpp
#include <vector>

class Item {
public:
    double getPrice() const {
        return price;
    }

    static double calculateTotal(const std::vector<Item>& items) {
        double total = 0;
        for (const Item& item : items) {
            total += item.getPrice();
        }
        return total;
    }

private:
    double price;
};

class OrderProcessor {
public:
    void processOrder() {
        double total = Item::calculateTotal(items);
        // Further order processing
    }

private:
    std::vector<Item> items;
};
